sdk/keyvault/azure-keyvault-secrets/azure/keyvault/secrets/models.py

```diff
@@ -163,27 +163,39 @@ def tags(self):
         return self._tags
 
 
-class Secret(SecretAttributes):
-    """All a secret's attributes, and its value."""
+class Secret(object):
+    """All of a secret's properties, and its value."""
 
-    def __init__(self, attributes, vault_id, value, **kwargs):
-        super(Secret, self).__init__(attributes, vault_id, **kwargs)
+    def __init__(self, properties, value, **kwargs):
+        self._properties = properties
         self._value = value
 
     @classmethod
     def _from_secret_bundle(cls, secret_bundle):
         # type: (_models.SecretBundle) -> Secret
         """Construct a Secret from an autorest-generated SecretBundle"""
         return cls(
-            secret_bundle.attributes,
-            secret_bundle.id,
-            secret_bundle.value,
-            content_type=secret_bundle.content_type,
-            key_id=secret_bundle.kid,
-            managed=secret_bundle.managed,
-            tags=secret_bundle.tags,
+            properties=SecretProperties(
+                attributes=secret_bundle.attributes,
+                vault_id=secret_bundle.id,
+                content_type=secret_bundle.content_type,
+                key_id=secret_bundle.kid,
+                managed=secret_bundle.managed,
+                tags=secret_bundle.tags
+            ),
+            value=secret_bundle.value
         )
 
+    @property
+    def properties(self):
+        # type: () -> SecretProperties
+        """
+        The secret properties
```

Comment:
Implies the properties are themselves secret. You probably want "the secret's properties".

Reply:
makes sense, switching the comment